src/vcpkg/registries.cpp

```diff
@@ -481,7 +481,7 @@ namespace
             return (*pscf)->to_version();
         }
         print_error_message(maybe_scf.error());
```

Comment:
I would do: 
```suggestion
        Debug::print(maybe_scf.error());
```
other suggestions?